custom_components/ocpp/api.py

```diff
@@ -1000,21 +1008,28 @@ def average_of_nonzero(values):
     @on(Action.MeterValues)
     def on_meter_values(self, connector_id: int, meter_value: dict, **kwargs):
         """Request handler for MeterValues Calls."""
-        self._metrics[csess.transaction_id.value].value = kwargs.get(
-            om.transaction_id.name, 0
-        )
+
+        transaction_id: int = kwargs.get(om.transaction_id.name, 0)
+
+        transaction_matches: bool = False
+        if transaction_id > 0:
```

Comment:
Can this if be removed because if transaction_id=0 if on 1016 will still evaluate to false?